stdlib/logging/__init__.pyi

```diff
@@ -63,10 +63,13 @@ if sys.version_info >= (3, 11):
 if sys.version_info >= (3, 12):
     __all__ += ["getHandlerByName", "getHandlerNames"]
 
+class _FilterProtocol(Protocol):
```

Comment:
You could rename this to `_SupportsFilter`. We have various other protocols named `SupportsX` that we use when we need a method named `X`. I'm not sure which way it's more readable though, so I'll let you decide :)

Reply:
Ah ok, I wasn’t sure of your naming conventions.